core/src/sumeragi/mod.rs

```diff
@@ -29,146 +27,25 @@ pub mod message;
 pub mod network_topology;
 pub mod view_change;
 
-use parking_lot::Mutex;
-
 use self::{message::*, view_change::ProofChain};
 use crate::{kura::Kura, prelude::*, queue::Queue, EventsSender, IrohaNetwork, NetworkMessage};
 
-/*
-The values in the following struct are not atomics because the code that
-operates on them assumes their values does not change during the course of
-the function.
-*/
-#[derive(Debug)]
-struct LastUpdateMetricsData {
-    block_height: u64,
-}
-
 /// Handle to `Sumeragi` actor
 #[derive(Clone)]
 pub struct SumeragiHandle {
-    state: Arc<State>,
-    metrics: Metrics,
-    last_update_metrics_mutex: Arc<Mutex<LastUpdateMetricsData>>,
-    network: IrohaNetwork,
-    kura: Arc<Kura>,
-    queue: Arc<Queue>,
+    /// Counter for amount of dropped messages by sumeragi
+    dropped_messages_metric: iroha_telemetry::metrics::DroppedMessagesCounter,
```

Comment:
seems like a useless thing to measure, I'd remove it

Reply:
We can detect that something is wrong with sumeragi, by observing this metric.
But i didn't have particular opinion on this metric.

Reply:
@0x009922 your thoughts?

Reply:
Seems like it might be useful, but I don't have a strong opinion on that neither.

Reply:
I would prefer to remove it in separate ticket if we decided to.
Since currently we don't break the API in any way.